circuits/test/bitreversal.test.ts

```diff
@@ -13,19 +13,13 @@ describe("bitreversal", () => {
   });
 
   let bit_array = [1,0,0,0,0,0,0,0];
-  let bit_array_1 = [1,0,0,0,0,0,0,1];
+  let expected_output = [0,0,0,0,0,0,0,1];
   it("should have correct output", async () => {
-    const witness = await circuit.calculateWitness({ in: bit_array });
+    const witness = await circuit.expectPass({ in: bit_array}, { out: expected_output });
     circuit.expectPass({in: bit_array});
```

Comment:
I recommend that tests get back the output and compare that to expected output instead of using `expectPass` to be consistent with the expressiveness of most testing frameworks.
``` js
const expect = ...
let result = await circuit.compute(input, ["out"]);
assert.equal(result, expected);
```
If not, remove the second `circuit.expectPass` and remove printing out the `witness` below.